packages/components/src/icons/EmbedIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+export const EmbedIcon = () => {
+  return (
+    <svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All those transforms counteract each other, you can optimize that:
```
<svg width="32px" height="32px" viewBox="0 0 32 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
  <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
    <polygon points="4 4 28 4 28 28 4 28"></polygon>
    <path
      d="M9.42130253,10.2684345 C10.3266052,9.36313187 11.7034439,10.7399706 10.7981413,11.6452732 L6.35806996,16.0849552 L10.7981413,20.5250265 C11.7034439,21.4303291 10.3266052,22.8071678 9.42130253,21.9018652 L4.29281186,16.7733745 C3.91277945,16.3933421 3.91277945,15.7765682 4.29281186,15.3965358 L9.42130253,10.2684345 Z M17.0772424,7.74721537 C17.4074345,6.51211003 19.2857914,7.01401759 18.9555993,8.2495123 L14.6222174,24.4234737 C14.2920253,25.658579 12.4136684,25.1566715 12.7438605,23.9211768 L17.0772424,7.74721537 Z M20.9017079,11.6452732 C19.9964053,10.7399706 21.373244,9.36313187 22.2785466,10.2684345 L27.4070373,15.3965358 C27.7870697,15.7765682 27.7870697,16.3933421 27.4070373,16.7733745 L22.2785466,21.9018652 C21.373244,22.8071678 19.9964053,21.4303291 20.9017079,20.5250265 L25.3417792,16.0849552 L20.9017079,11.6452732 Z"
      fill="#191919"
      fillRule="nonzero"
    ></path>
  </g>
</svg>
```

Reply:
Good catches thanks, was just copy-pasting svg assets. Strange that `fill-rule` wasn't caught though I seem to remember some part of linting catching malformed names.